src/components/config/validation/validator/probe.ts

```diff
@@ -160,6 +160,7 @@ export async function validateProbes(probes: Probe[]): Promise<Probe[]> {
             .min(1),
           interval: joi.number().default(DEFAULT_INTERVAL).min(1),
           lastEvent: joi.object({
+            alertId: joi.string(),
             createdAt: joi.string().allow(''),
             recoveredAt: joi.string().allow('', null),
           }),
```

Comment:
**Suggestion:** Consider adding a validation rule for the 'alertId' field to ensure it's not an empty string.
```suggestion
          lastEvent: joi.object({
            alertId: joi.string().min(1),
            createdAt: joi.string().allow(''),
            recoveredAt: joi.string().allow('', null),
          }),
```